### PR TITLE
Create first capstone project africa ai expo

### PR DESCRIPTION
**Pull request by Lembani**
### This PR is for the first capstone project. The project name is Africa AI Expo.

[Here is a link to the project documentation video](https://www.loom.com/share/3a23a8af2b5244da93529458d7855e26)

This project has the following for both Mobile and Desktop:
1. Home page
- Header section
- Main program section
- Speaker section (Dynamically rendered)
- Footer section

2. About page
- Header section
- Logo section
- Past Expo's section
- Footer section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+    <section class="program-section" id="program">
+        <h2 class="program-heading">Main Program</h2>
+        <hr class="program-line">
+        <div class="program-container">
+            <div class="program-item">
+                <img src="./images/program1.png" alt="Lecture" class="program-icon">
+                <h4 class="title">Speak</h4>
+                <p class="program-description">Lorem ipsum dolor sit amet consectetur.</p>
+            </div>
```
Comment (Cathella) [inline index.html]: - [x] To make the code more semantic, I would suggest making the `div` wrappers with the program-item class changed to `article` tags

```
<article class="program-item"></article>
```

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @Lembani 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Descriptive PR and README.
✅ Used GitHub Pages to deploy the project.
✅ Project well styled as seen in Behance design


## Required Changes ♻️
_Check the comments under the review._

- [x] When I hover over the link in the mobile nav, the link becomes invisible, this is because the onHover link color is the same as the background color of the nav. Kindly change the link color.

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Lembani) [reply index.html]: Hi @Cathella, Thanks for the review. 
I have fixed the issues above. Kindly review them before I submit another request for review.
Regards.

**Review by phelian23 — APPROVED** [review]
Hi @Lembani, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Cheers and Happy coding!👏👏👏

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**